docs/reference/chisel-releases/chisel.yaml.md

```diff
@@ -0,0 +1,309 @@
+packages for. This value is currently only used for logging, and does not change
+the archive behaviour.
+
+
+(chisel_yaml_format_spec_archives_suites)=
+### `archives.<name>.suites`
+
+| Field    | Type        | Required | Supported values                                              |
+| -------- | ----------- | -------- | ------------------------------------------------------------- |
+| `suites` | `list(str)` | Required | Ubuntu archive suite names e.g. `jammy`, `noble-updates` etc. |
+
+Lists the archive suites to fetch packages from. Read more
+about suites in the [Ubuntu packaging
+guide](https://canonical-ubuntu-packaging-guide.readthedocs-hosted.com/en/latest/explanation/archive/#suite).
+
+
+(chisel_yaml_format_spec_archives_components)=
+### `archives.<name>.components`
+
+| Field        | Type        | Required | Supported values                                   |
+| ------------ | ----------- | -------- | -------------------------------------------------- |
+| `components` | `list(str)` | Required | Suite component names e.g. `main`, `universe` etc. |
+
+Lists the components of the archive suites to fetch
+packages from. Read more about components in the [Ubuntu packaging
+guide](https://canonical-ubuntu-packaging-guide.readthedocs-hosted.com/en/latest/explanation/archive/#components).
+
+Chisel reads the `InRelease` files from each `(suite, component)` combination to
+locate packages.
+
+
+(chisel_yaml_format_spec_archives_public_keys)=
+### `archives.<name>.public-keys`
+
+| Field         | Type        | Required | Supported values                                                            |
+| ------------- | ----------- | -------- | --------------------------------------------------------------------------- |
+| `public-keys` | `list(str)` | Required | List of key names, as defined in {ref}`chisel_yaml_format_spec_public_keys` |
+
+Lists the names of the OpenPGP public keys needed to verify the archive's `InRelease`
+file signatures. These key names must be defined in
+{ref}`chisel_yaml_format_spec_public_keys`.
+
+
+(chisel_yaml_format_spec_archives_priority)=
+### `archives.<name>.priority`
+
+| Field      | Type  | Required                                         | Supported values                   |
+| ---------- | ----- | ------------------------------------------------ | ---------------------------------- |
+| `priority` | `int` | Required with multiple archives, if no `default` | Any integer between -1000 and 1000 |
+
+Describes the priority of an archive compared to other
+archives. It is used to support multiple archives in Chisel. If a package is
+available in two archives, it is fetched from the archive with higher priority,
+unless:
+ - the package's slice definitions file specifies {ref}`"archive"<slice_definitions_format_archive>`, or
+ - the {ref}`chisel_yaml_format_spec_archives_default` field is `true` for any archive.
+In this case, the priority values of **all** archives are ignored.
```

Comment:
This is not true, if priorities are set then the default is ignored in new Chisel versions. For Chisel <= 1.0 it cannot read priorities so obviously default is used. Maybe we can say "introduced in version x" when talking about a feature.

Reply:
This is indeed what I thought and we discussed offline. Patched in f77e94b